### ISSUE TITLE
Version control file status colors are reverted after a while

### ISSUE THREAD

**Issue by sebastiaanluca** [ChrisRM/material-theme-jetbrains#1793]
### Summary

Version control file status colors are reverted after a while.

### Expected behavior

Disabled file status colors remain disabled.

### Actual behavior

File status colors are reverted after a while, seemingly randomly after pushing to a remote git repository from another app.

### Steps to reproduce (if needed)

1. Go into preferences > version control > file status colors > disable each one

![Screenshot 2021-07-09 at 15 05 41](https://user-images.githubusercontent.com/711940/125082549-a7026d80-e0c7-11eb-8d42-ca759eac6859.jpg)

I don't have a subscription, yet the material theme file status colors option is enabled. I can't disable it, so not sure if it's active. However, only the version control colors seem to have effect.

![Screenshot 2021-07-09 at 15 06 03](https://user-images.githubusercontent.com/711940/125082713-d6b17580-e0c7-11eb-9dd4-930611f779b0.jpg)

### Environment

PhpStorm 2021.1.4
Build #PS-211.7628.25, built on June 30, 2021
Runtime version: 11.0.11+9-b1341.60 aarch64
VM: OpenJDK 64-Bit Server VM by JetBrains s.r.o.
macOS 11.4
GC: ParNew, ConcurrentMarkSweep
Memory: 12221M
Cores: 8
Registry: editor.show.special.chars=false, ide.mac.message.dialogs.as.sheets=false, bigger.font.in.project.view=true, search.everywhere.toolwindows=false, run.processes.with.pty=TRUE, search.everywhere.configurations=false, ide.allow.merge.buttons=false, ide.tooltip.initialDelay=0, ide.balloon.shadow.size=0, ide.images.show.chessboard=true
Non-Bundled Plugins: Shifter (1.9.6), com.mallowigi (36.0), org.sylfra.idea.plugins.linessorter (1.0.1), some.awesome (1.14), com.kalessil.phpStorm.phpInspectionsEA (4.0.6.4), dev.nybroe.collector (0.3.2), intellij.prettierJS (211.7142.13), com.chrisrm.idea.MaterialThemeUI (6.7.2), ru.adelf.idea.dotenv (2021.2)

**Comment by mallowigi** [comment]
I'll fix the issue with the checkbox being disabled for free users, but for the rest, please read this:

https://www.material-theme.com/docs/configuration/file-status-colors/#material-file-colors

TLDR: do not use the `File Status Colors` page

**Comment by sebastiaanluca** [comment]
Thank you!
